app/resources/views/partials/talk/joindin.html.twig

```diff
@@ -0,0 +1,5 @@
+{% if talk.joindinUrl %}
+    <div style="text-align:right;">
```

Comment:
There should be a `text-right` class for this.

Reply:
I was having issues yesterday with some of the inbuilt class for tailwindcss not showing up or working (I had the same issue with h-full). Seems to not be a problem now though, so I must have done something!